src/modules/geospatial/dto/get-address-by-postcode-query.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { GEOSPATIAL } from '@ukef/constants';
+import { Matches, MaxLength, MinLength } from 'class-validator';
+
+const UK_POSTCODE = /^[A-Za-z]{1,2}[\dRr][\dA-Za-z]?\s?\d[ABD-HJLNP-UW-Zabd-hjlnp-uw-z]{2}$/;
```

Comment:
What is the source for this RegExp?

Reply:
Regex is from DTFS project and slightly optimized by lint.

Now I moved it to constants and left comment about source.